WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -95,4 +128,35 @@
   });
 
   $('#display-<%= @display %>').button('toggle');
+
+  // From http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/901144#901144
+  function getParameterByName(name, url) {
```

Comment:
Actually, in light of my later comments, I'm pretty sure we can get rid of this method entirely.